src/components/Definitions/Definitions.scss

```diff
@@ -20,10 +20,43 @@ limitations under the License.
 }
 
 main {
+
   .bx--structured-list {
-    background-color: white;
+    background-color: $ui-background;
   }
-}
+
+  .bx--structured-list-row:nth-child(odd) {
+      background-color: white;
+  }
+
+  .bx--structured-list-row:nth-child(even) {
+    background-color: $ui-01;
+  }
+
+  .bx--modal.is-visible {
```

Comment:
Please remove the style overrides for the modal buttons / footer. Primary button colour will be blue/red depending on type of modal and secondary button will usually be black. These are handled by Carbon and are consistent with other modals in the UI, including the experimental repo which was recently updated by @carlos-logro to follow these design guidelines.